dev/com.ibm.ws.jsonp.2.0_fat/fat/src/com/ibm/ws/jsonp20/fat/FATSuite.java

```diff
@@ -0,0 +1,24 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jsonp20.fat;
```

Comment:
instead of creating a new FAT bucket, I would prefer that we add the Jakarta repeat rules to existing buckets as described here:
https://github.com/OpenLiberty/open-liberty/wiki/FAT-tests#repeating-fat-tests-for-jakarta-ee-9

Reply:
The jsonp1.1_fat is not ee9 repeatable.